examples/client/client.c

```diff
@@ -2122,6 +2123,20 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
                 #endif
                 break;
 
+            case '@' :
+            {
+                const char *conf_args = libwolfssl_configure_args();
```

Comment:
This appears to break on Windows where ./configure is not used... Maybe wrap all this with HAVE_CONFIG_H?

Reply:
Yeah no doubt -- a bunch of gating is coming.  First step is to restore my build-time gating on debug.c to src/install.am, and compile in the config strings only when examples are being built.